src/utils/hints/contractsHints.js

```diff
@@ -3,4 +3,18 @@ export const contractsHints = {
   contractCreated: 'Keyblock height and exact date and time when the smart contract was created.',
   contractTxHash: 'The transaction hash of a ContractCreateTx that was executed to create the smart contract on the blockchain.',
   contractCreator: 'Account that created the smart contract.',
+  bytecode: 'Low-level representation of the smart contract that is generated when the contract\'s source code is compiled.',
+  contractsAccount: 'The address where the smart contract is located. To be used if a smart contract is already deployed.',
+  contractsAccountBalance: 'Smart contract account balance that is needed to execute "payable" functions.',
+  contractCalls: 'Amount of calls to the smart contract.',
+  apiLinks: 'Links to sources of the smart contract data.',
+  callTransactionsHash: 'Low-level representation of the smart contract that is generated when the contract\'s source code is compiled.',
+  callTransactionsTime: 'Estimated date, time and block height when the smart contract was called.',
+  callTransactionsCaller: 'Account that called the smart contract.',
+  callTransactionsEntryPoint: 'Name of the smart contract function that was called.',
+  eventsCallTransaction: 'Unique identifier of the ContractCallTx transaction which emitted the event.',
+  eventsCreated: 'Estimated date, time and block height when the smart contract had emitted the event.',
+  eventsName: 'Name of the emitted event.',
+  eventsArguments: 'The arguments of the emitted event.',
```

Comment:
I guess these are the values (data) emitted for the event? what format is this?

Reply:
@michele-franchi  ?

Reply:
Not sure if I got your question but these are encoded arguments.

Reply:
to me this is a bit strange displayed. I assume these can be different (encoded) values. I am also not sure if I would replace "arguments" with "values". right now it is not clear if this is only one value or whether these can be different values. or is it one combined encoded value for all possible values/arguments ? 😅 

the thing is that it is cutted and can be copied, so the user doesn't really see the value(s). also, they are quite meaningless anyway at this point of time.

Reply:
Well, I clarified this with Rogerio and I would do the following:
- Get rid of the "Data" column as it's a really specific use case
- Rename arguments column to "Data" to be consistent with [Token events](https://aescan.dev.aepps.com/tokens/ct_KeTvHnhU85vuuQMMZocaiYkPL9tkoavDRT3Jsy47LK2YqLHYb)

In a separate issue:
- Decode known args (using the parameter aexn-args=true)
- Find a better way to print arguments that cannot be decoded? In my opinion in any case these should be printed somehow and explain to the user what should be done in order to decode them

Reply:
@michele-franchi what you mean by `to be consistent with Token events` ? This issue is about Token events.

Reply:
Ah, sorry this is about contract events, got it .) Will fix it

Reply:
fixed, please check now

Reply:
Followup https://github.com/aeternity/aescan/issues/362